src/Kiota.Builder/Refiners/CommonLanguageRefiner.cs

```diff
@@ -174,16 +156,25 @@ public abstract class CommonLanguageRefiner : ILanguageRefiner
             CrawlTree(current, x => AddConstructorsForDefaultValues(x, addIfInherited));
         }
         protected static void ReplaceReservedNames(CodeElement current, IReservedNamesProvider provider, Func<string, string> replacement, HashSet<Type> codeElementExceptions = null) {
-            if(current is CodeClass currentClass && currentClass.StartBlock is CodeClass.Declaration currentDeclaration)
-                currentDeclaration.Usings
-                                    .Select(x => x.Declaration)
-                                    .Where(x => x != null && !x.IsExternal)
-                                    .Join(provider.ReservedNames, x => x.Name, y => y, (x, y) => x)
-                                    .ToList()
-                                    .ForEach(x => {
-                                        x.Name = replacement.Invoke(x.Name);
-                                    });
-
+            if(current is CodeClass currentClass && currentClass.StartBlock is Declaration currentDeclaration)
```

Comment:
Maybe the first if statement should also maybe check the `codeElementExceptions` collection? I'm not sure if that will ever be needed but might be useful to add a `CodeUsing` to the blacklist.

Reply:
fixed via be50e85 (push pending)

Reply:
Sorry. Can we add `CodeUsing` to the blacklist as well? 
I just noticed this will be capitalized as well so we don't need to worry about it for dotnet. 
This will make namespace names a bit cleaner too instead of having something like `ApiSdk.Applications.Item.CreatedOnBehalfOf.@Ref` for a namespace name.

Reply:
Yes. Exactly.

Reply:
thanks. solved.